app/code/Bulbulatory/Recommendations/Helper/Email.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+namespace Bulbulatory\Recommendations\Helper;
+
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\App\Helper\Context;
+use Magento\Framework\Mail\Template\TransportBuilder;
+use Magento\Framework\Translate\Inline\StateInterface;
+use Magento\Store\Model\StoreManagerInterface;
+
+/**
+ * Custom Module Email Helper
+ */
+class Email extends AbstractHelper
+{
+    const XML_PATH_EMAIL_TEMPLATE_FIELD  = 'recommendations/general/recommendation_email';
+
+    /**
+     * @var ScopeConfigInterface
+     */
+    protected $_scopeConfig;
+
+    /**
+     * Store manager
+     *
+     * @var StoreManagerInterface
+     */
+    protected $_storeManager;
+
+    /**
+     * @var StateInterface
+     */
+    protected $inlineTranslation;
+
+    /**
+     * @var TransportBuilder
+     */
+    protected $_transportBuilder;
+
+    /**
+     * @var string
+     */
+    protected $temp_id;
+
+    /**
+     * @param Context $context
+     * @param StoreManagerInterface $storeManager
+     * @param StateInterface $inlineTranslation
+     * @param TransportBuilder $transportBuilder
+     */
+    public function __construct(
+        Context $context,
+        StoreManagerInterface $storeManager,
+        StateInterface $inlineTranslation,
+        TransportBuilder $transportBuilder
+    ) {
+        $this->_scopeConfig = $context;
+        parent::__construct($context);
+        $this->_storeManager = $storeManager;
+        $this->inlineTranslation = $inlineTranslation;
+        $this->_transportBuilder = $transportBuilder;
+    }
+
+    /**
+     * Return store configuration value of your template field that which id you set for template
+     *
+     * @param string $path
+     * @param int $storeId
+     * @return mixed
+     */
+    protected function getConfigValue($path, $storeId)

```

Comment:
It's more readable and maintainable if you split the configuration logic to another helper, put all the configuration related logic there and inject it in this helper (which will have only email related methods). There may be some other places that will need to use the configuration so by creating a config helper it will be possible to use it there, without all the email stuff that may not be needed.

Reply:
@TM18

> It's more readable and maintainable if you split the configuration logic to another helper, put all the configuration related logic there and inject it in this helper (which will have only email related methods). There may be some other places that will need to use the configuration so by creating a config helper it will be possible to use it there, without all the email stuff that may not be needed.

Changed! I am create **Bulbulatory\Recommendations\Helper\Config** class.